app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person_info in people:
+        person = Person(person_info["name"], person_info["age"])
+        if "wife" in person_info and person_info["wife"] is not None:
+            wife_person = Person.people.get(person_info["wife"])
+            if wife_person:
+                person.wife = wife_person
+                wife_person.husband = person
+
+        elif "husband" in person_info and person_info["husband"] is not None:
+            husband_person = Person.people.get(person_info["husband"])
+            if husband_person:
+                person.husband = husband_person
+                husband_person.wife = person
```

Comment:
> this 2 blocks of code is similar besides the wife/husband string, try to optimize ur solution
> hint: u can iterate through ['wife', 'husband'] list, and use this str as keys

I see you didn't make changes that Yuri requested from previous review. Please simplify